lib/postmark/mail_message_converter.rb

```diff
@@ -30,7 +30,8 @@ def headers_part
         'Subject' => @message.subject,
         'Headers' => @message.export_headers,
         'Tag' => @message.tag.to_s,
-        'TrackOpens' => cast_to_bool(@message.track_opens)
+        'TrackOpens' => cast_to_bool(@message.track_opens),
+        'TrackLinks' => @message.track_links
```

Comment:
Seems weird that `track_links` doesn’t use `cast_to_bool` as well. I’d prefer to keep things consistent.